linters/detekt/plugin.yaml

```diff
@@ -11,6 +11,7 @@ lint:
     - name: detekt
       files: [kotlin]
       download: detekt
+      description: Static code analysis for Kotlin
```

Comment:
Can we add this to the other `detekt` linters as well (detekt-explicit, detekt-gradle)